pages/common/jenkins.md

```diff
@@ -0,0 +1,24 @@
+# jenkins-cli
+
+> An open-source automation server that facilitates the automation of software development lifecycle.
+> More information: <https://www.jenkins.io/doc/>.
+
+- Connect to jenkins CLI:
+
+`java -jar jenkins-cli.jar -s {{jenkins_server_url}} -auth {{username}}:{{api_token}}`
+
+- Display help:
+
+`java -jar jenkins-cli.jar -s {{jenkins_server_url}} help`
+
+- Display Version:
+
+`java -jar jenkins-cli.jar -s {{jenkins_server_url}} version`
```

Comment:
Move help and version as the last commands on the page https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#help-and-version-commands